app/client/views/admin/user/user.html

```diff
@@ -55,10 +55,6 @@
         <input type="text" ng-model="selectedUser.email" />
       </div>
 
-      <div class="field">
```

Comment:
we might want to make this a devpost link in the future, but good to remove it for now